### ISSUE TITLE
oe_content_event_event_programme 

### ISSUE THREAD

**Issue by augustabande** [openeuropa/oe_theme#1082]
In the module **OpenEuropa Theme Event Content** I have the _oe_content_event_event_programme_ missing. Any idea of why and what I can do in order to install it?

**Comment by sergepavle** [comment]
Please, try to run the composer update. Seems like openeuropa/oe_content package is not up to date.

**Comment by p4r0d5** [comment]
@sergepavle  
if I create the site from the scratch I receive the error 
oe_content_event_event_programme (missing) in the OpenEuropa Theme Event Content module.
please have a look at : openeuropa/oe_theme/blob/3.x/modules/oe_theme_content_event/oe_theme_content_event.post_update.php
line96
(composer update does not fix it)

**Comment by sergepavle** [comment]
Please, make sure that `openeuropa/oe_content` package is installed by the composer and within ^3.0.0 constrain. This problem also can be when you run composer update with '--no-dev' option. For your site this `openeuropa/oe_content` package should be present in `require` section of composer.json.

**Comment by p4r0d5** [comment]
you are right, I didnt set the correct constrain in the composer file, thanks so much.
